invenio_mail/templates/invenio_mail/base.txt

```diff
@@ -0,0 +1,37 @@
+{#
+# This file is part of Invenio.
+# Copyright (C) 2016 TIND.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#}
+{%- block header -%}
```

Comment:
As this is a text email, correct whitespace is quite important. Can you check how an email rendered with this template looks like, and ensure that there's no excessive whitespace (e.g. in the header or between greeting and content.